examples/flower-in-30-minutes/tutorial.ipynb

```diff
@@ -826,23 +790,47 @@
    "execution_count": null,
    "metadata": {
     "colab": {
-     "base_uri": "https://localhost:8080/",
-     "height": 508
+     "base_uri": "https://localhost:8080/"
     },
-    "outputId": "d8eab106-cee9-4266-9082-0944882cdba8"
+    "outputId": "9ad8dcea-8004-4c6e-a025-e168da636c88"
    },
    "outputs": [],
    "source": [
-    "print(f\"{history.metrics_centralized = }\")\n",
+    "from flwr.server import ServerApp\n",
+    "from flwr.simulation import run_simulation\n",
     "\n",
-    "global_accuracy_centralised = history.metrics_centralized[\"accuracy\"]\n",
-    "round = [data[0] for data in global_accuracy_centralised]\n",
-    "acc = [100.0 * data[1] for data in global_accuracy_centralised]\n",
-    "plt.plot(round, acc)\n",
-    "plt.grid()\n",
-    "plt.ylabel(\"Accuracy (%)\")\n",
-    "plt.xlabel(\"Round\")\n",
-    "plt.title(\"MNIST - IID - 100 clients with 10 clients per round\")"
+    "server_app = ServerApp()\n",
+    "\n",
+    "\n",
+    "@server_app.main()\n",
+    "def main(driver, context):\n",
+    "    # Construct the LegacyContext\n",
+    "    context = fl.server.LegacyContext(\n",
```

Comment:
I encountered the error `Exception ClientAppException occurred. Message: 'str' object has no attribute 'node_config'` when I ran this cell. I haven't debugged it yet, but it might need to be modified.

Reply:
I think you have the old Flower installed. `node_config` is only available if you install Flower main.